tide/master/PixelStreamWindowManager.h

```diff
@@ -97,10 +105,10 @@ class PixelStreamWindowManager : public QObject
      * @param pos the desired position for the center of the window in pixels.
      *        If pos.isNull(), the window is centered on the DisplayGroup.
      * @param size the desired size of the window in pixels.
-     * @param webbrowser create a webbrowser window.
+     * @param StreamType enum controls the focus of a window upon opening. WEBBROWSER creates a webbrowser window.
```

Comment:
This reveals too much about the implementation, users should not rely on the internal behaviour. Could be put more simply:
@param stream the type of window to create